edgeless_api/src/function_instance/mod.rs

```diff
@@ -57,6 +59,21 @@ pub struct SpawnFunctionRequest {
     pub state_specification: StateSpecification,
 }
 
+#[derive(Debug, Clone)]
+pub struct SpawnFunctionResponse {
```

Comment:
Especially the rust representation must be a result-like object.

Reply:
OK, I can agree to that (my idea was to keep the Rust structures close to the original gRPC data structures, but there is no need for this).

Reply:
As suggested above, the Rust trait is the reference, the gRPC binding only one of possibly many bindings.

Reply:
Fine, will do that. I am not opening an issue for this, it will part of the upcoming API refactoring.